DDD.Functions/local.settings.json

```diff
@@ -4,6 +4,7 @@
     // Web Jobs
     "AzureWebJobsStorage": "UseDevelopmentStorage=true",
     "AzureWebJobsDashboard": "UseDevelopmentStorage=true",
+    "FUNCTIONS_WORKER_RUNTIME": "dotnet",
```

Comment:
Didn't know this. Nice.

Reply:
Yeah the local emulator told me to do it. It's a recent addition. It makes startup of the local emulator way faster and less errorfull because it doesn't try to execute the java and javascript engines.